tools/importer-rest-api-specs/components/parser/operations.go

```diff
@@ -5,6 +5,7 @@ package parser
 
 import (
 	"fmt"
+	"github.com/hashicorp/pandora/tools/data-api-sdk/v1/models"
```

Comment:
imports need sorting here

Reply:
This is fixed in one of the later PRs fwiw